master/internal/api_trials.go

```diff
@@ -109,8 +109,8 @@ func (a *apiServer) TrialLogs(
 	terminateCheck := api.TerminationCheckFn(func() (bool, error) {
 		state, endTime, err := a.m.db.TrialStatus(int(req.TrialId))
 		if err != nil ||
-			// Give trials a moment to finish sending logs after termination.
-			(model.TerminalStates[state] && endTime.After(time.Now().Add(-10*time.Second))) {
+			// Give trials a bit to finish sending logs after termination.
```

Comment:
TIL 1 bit = 3 moments. :wink:

Reply:
Lol, felt like i couldnt rightly call it a moment still. Glad you noticed.